site/en/blog/new-in-chrome-114/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: New in Chrome 114
+description: >
+  Chrome 114 is rolling out now! Chrome 114 is rolling out now! With text-wrap: balance to improve text layouts, Cookies Having Independent Partitioned State are here, the new Popover API makes popovers easier than ever, and there’s plenty more.
+layout: 'layouts/blog-post.njk'
+date: 2023-05-30
+authors:
+  - ajara
+hero: 'image/SeARmcA1EicLXagFnVOe0ou9cqK2/hY1Jhm9rglpkrvoegKyu.jpg'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-114
+---
+
+{% YouTube id='qqTnAWL7-v0' %}
+
+Here's what you need to know:
+
+* CSS [`text-wrap: balance`] (#text-wrap-balance) is available to improve text layouts.
+* Cookies partitioned by top level site ([CHIPS](#chips))are here.
+* Popovers are easier than even with the [Popover API](#popover-api).
+* And there’s plenty [more](#more).
+
+I’m Adriana Jara. Let’s dive in and see what’s new for developers in Chrome 114.
+## `text-wrap:balance`. {: #text-wrap-balance}
+Use `text-wrap-balance` to improve text layouts. The animation below shows the difference you can make with this one line.
+
+<figure>
+  {% Video
+    src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/qJKWQGssebOIDGVBtLpo.mp4",
+    autoplay="true",
+    loop="true",
+    muted="true",
+    controls="true"
+  %}
+
+  <figcaption>
+    <a href="https://codepen.io/web-dot-dev/pen/KKxjpQm">
+      Try a demo
+    </a>
+  </figcaption>
+</figure>
+
+As a developer, you don’t know the final size, font size, or even language of text. All variables“are“needed for an effective treatment of text wrapping. Since the browser does know all the factors, with text-wrap:balance you can request the browser to figure out the best balanced line wrapping solution.
```

Comment:
```suggestion
As a developer, you don’t know the final size, font size, or even language of text. All variables are needed for an effective treatment of text wrapping. Since the browser does know all the factors, with text-wrap:balance you can request the browser to figure out the best balanced line wrapping solution.
```